datasets/bnl_newspapers/README.md

```diff
@@ -0,0 +1,199 @@
+---
+annotations_creators: []
+language_creators:
+- other
+languages:
+- ar
+- da
+- de
+- fi
+- fr
+- lb
+- nl
+- pt
+licenses:
+- cc0-1.0
+multilinguality:
+- multilingual
+pretty_name: bnl-processed-newspapers
```

Comment:
I find it prettier named this way:
```suggestion
pretty_name: BnL processed newspapers
```

Reply:
I've removed the 'processed' part but otherwise accepted this suggestion.